test/region_merging.jl

```diff
@@ -0,0 +1,59 @@
+acc_type{T}(a::T) = Images.accum(T)(a);
+homogeneous{T<:Union{Colorant,Real}, N}(i::AbstractArray{T,N}) = (acc_type(maximum(i))-acc_type(minimum(i)) < 1)
```

Comment:
Might be ever so slightly more efficient with `extrema`? I'm not sure that's true, but `extrema` only needs one trip through the array whereas this needs two.